supported-standards/lens-open-algorithm-standards/models/publication/feed/personal-schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://lens.xyz/post.feed.schema.json",
+  "title": "Post Feed",
+  "description": "Ordered list of posts to be shown to users",
+  "type": "object",
+  "properties": {
+    "profileId": {
+      "type": "string",
+      "description": "ProfileId for which the posts are being recommended",
+      "pattern": "^0x[0-9a-fA-F]+$"
+    },
+    "posts": {
```

Comment:
dont we want to call it publications?

Reply:
The main reason for specifying the type of publication is that we found that it is easier for clients that are consuming model results to know exactly the type of publication especially since Lens BigQuery Public dataset has separate tables for posts and comments. But, I can address this by adding type to the result tuple.